lib/core/network/protocols/httpwsProtocol.js

```diff
@@ -782,15 +779,6 @@ class HttpWsProtocol extends Protocol {
     // Send Headers in one go
     response.cork(() => {
       this.httpWriteRequestHeaders(request, response, message);
-
-      if (streamSizeFixed) {
-        response.writeHeader(
-          HTTP_HEADER_CONTENT_LENGTH,
-          Buffer.from(httpStream.totalBytes.toString())
-        );
```

Comment:
Don't you want to keep the Content-Length header management?

Reply:
No it's also set by uWebsocket when writing with `tryEnd`

Reply:
Ok didn't know about it 🙂